services/twitch.py

```
@@ -109,8 +173,32 @@ def process_clip(self, clip: TwitchClip) -> bool:
+                # Persist video file first
+                persistent_video_path = self.cache_manager.persist_file(video_path, video_filename, clip.created_at)
+                
+                # Persist audio file
+                persistent_audio_path = self.cache_manager.persist_file(audio_path, audio_filename, clip.created_at)
+                
+            except Exception:
+                # Clean up any successfully persisted files on failure
+                if persistent_video_path:
+                    try:
+                        self.cache_manager.delete_persisted_file(persistent_video_path)
+                    except Exception as cleanup_error:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)